src/data/polynomial/derivative.lean

```diff
@@ -177,6 +177,15 @@ begin
   exact ih _ this (this.trans_le $ nat.le_of_lt_succ hx) rfl
 end
 
+@[simp] lemma iterate_derivative_C {k} (h : 0 < k) : (derivative^[k] (C a : R[X])) = 0 :=
```

Comment:
Can `simp` solve inequalities like `0 < k` or `1 < k`, at least for concrete numerals? It not, these shouldn't be marked `simp`.

Reply:
I found it surprising as well, but these were already `simp` lemmas.